source/system/system.go

```diff
@@ -0,0 +1,46 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package system
+
+import (
+	"log"
+	"os"
+
+	"sigs.k8s.io/node-feature-discovery/source"
+)
+
+var logger = log.New(os.Stderr, "", log.LstdFlags)
```

Comment:
Similar to what I commented in #195, I think it's better just to use `log.Printf()` instead of initializing a logger with default settings.

Reply:
Done.